src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/DefaultRazorComponentSearchEngine.cs

```diff
@@ -86,17 +87,29 @@ public override async Task<DocumentSnapshot> TryLocateComponentAsync(TagHelperDe
             return null;
         }
 
-        public static bool IsPathCandidateForComponent(DocumentSnapshot documentSnapshot, string typeName)
+        private string RemoveGenericContent(string typeName)
+        {
+            var genericSeparatorStart = typeName.IndexOf('<');
```

Comment:
Any chance `typeName` is a tuple?

`(ClassA<T>, ClassB<T>)` -> `(ClassA`

Reply:
> Any chance `typeName` is a tuple?

Good idea but nope, not for components 😄